_config.yml

```diff
@@ -1,5 +1,26 @@
-name: Crystal Programming Language
-description: Crystal Programming Language's blog
+name: The Crystal Programming Language
+description: A compiled language with Ruby like syntax and type inference
```

Comment:
It doesn't have to be that short either btw, two to three sentences should be fine still.